src/main/resources/db/changelog/changesets/changelog_20241126T134816Z.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet author="phamquy (generated)" id="1732628915649-1">
+        <sqlFile
+                dbms="postgresql"
+                encoding="UTF-8"
+                path="migrationDataDynamicSimulationProvider.sql"
+                relativeToChangelogFile="true"
+                splitStatements="true"
+                stripComments="true"/>
```

Comment:
doing a sql file just for this is a little overkill, just let liquibase mange it
```suggestion
        <update tableName="study">
            <column name="dynamic_simulation_provider" value="Dynawo"/>
            <where>dynamic_simulation_provider='DynaWaltz'</where>
        </update>
```

Reply:
Nice! It works! I have never used liquibase for manipulating data..